spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -205,6 +206,19 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
   }
 
   /**
+   * set trainable. If trainable is false, its parameters(weight/bias, if exists)
+   * are not changed in training process
+   * @param trainable whether this layer is trainable
+   */
+  def setTrainable(trainable: Boolean): this.type = {
```

Comment:
but if setTrainable is true, the user may expect that the layer become trainable again, but the method doesn't, right?